pom.xml

```diff
@@ -22,7 +22,8 @@
     <revision>6.19</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.222.4</jenkins.version>
+    <!-- remember to change the io.jenkins.tools.bom artifact when changing this -->
```

Comment:
True of any plugin but :shrug:.

(I wonder if we can create some custom Enforcer rule for this? Would get sticky when `bom-weekly` is involved.)

Reply:
Depending on weeklies, until the corresponding LTS base is selected, is fine when there is a particular reason for it (as there often has been lately). Anyway the more common case is you are just moving from one LTS dep to a newer and forgot to switch the BOM import.

Reply:
> the recent changes I have seen are plugin breakages (Guava)

Can happen but mostly the plugin is not exactly broken, just including redundant or deprecated stuff that can now be cleaned up significantly. There are also significant changes to test systems, alignment with UI changes, taking advantage of better APIs to offer comparable functionality with cleaner code, etc.